src/commands.ts

```diff
@@ -101,7 +103,7 @@ export class Commands {
 			console.error(e);
 			return;
 		}
-
+		const encryptDDO = args[2] === "false" ? false : true;
```

Comment:
Can you validate that args[2] is not null first? If it is null, this call will crash